packages/rancher-kiali-server/rancher-kiali-server.patch

```diff
@@ -1,7 +1,7 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/Chart.yaml packages/rancher-kiali-server/charts/Chart.yaml
 --- packages/rancher-kiali-server/charts-original/Chart.yaml
 +++ packages/rancher-kiali-server/charts/Chart.yaml
-@@ -1,20 +1,28 @@
+@@ -1,20 +1,26 @@
```

Comment:
Do we need this change as well or should this be removed like `rancher-monitoring`?

Reply:
good catch! I removed this change as well.